app/models/subscriber.rb

```diff
@@ -4,8 +4,6 @@ class Subscriber < ApplicationRecord
     validates :address, uniqueness: { case_sensitive: false }
   end
 
-  validate :not_nullified_and_activated
```

Comment:
Minor: it was puzzling me why this was the first commit, so you could explain it's required for the next one (where we nullify without deactivating).

Reply:
I've combined this commit into the one after so there isn't the initial confusion. 😄